src/util/helpers.py

```diff
@@ -339,3 +339,25 @@ def _remove_export_lines(cr, model, field=None):
     ]
     if to_rem:
         cr.execute("DELETE FROM ir_exports_line WHERE id IN %s", [tuple(to_rem)])
+
+
+def _remove_import_mappings(cr, model, field=None):
```

Comment:
Following @andreabak suggestion I think you can just merge these two into one `_remove_import_export_lines`